components/layout/table.rs

```diff
@@ -230,7 +230,7 @@ impl Flow for TableFlow {
         // part of the table, which we don't want to do—it belongs to the table wrapper instead.
 
         // Get column inline sizes from colgroups
-        for kid in self.block_flow.base.child_iter().filter(|kid| kid.is_table_colgroup()) {
+        for kid in self.block_flow.base.child_iter_mut().filter(|kid| kid.is_table_colgroup()) {
```

Comment:
Because this is a different function altogether; `flow::child_iter_mut()` operates on Flows, and this `child_iter_mut()` is being called on a different struct.